.github/workflows/docker.yml

```diff
@@ -0,0 +1,29 @@
+name: Docker
+
+on:
+  pull_request:
+    branches: [main]
+
+jobs:
+  build:
+    name: Build and Run
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          submodules: recursive
+
+      - name: Run the Docker image
+        run: |
+          mkdir kb
+          DOCKER_BUILDKIT=1 docker build . -t ubuntu/sc-machine:local
+          docker run --rm -e REBUILD_KB=1 -v ${ls}/kb:/kb -v kb_binary:/kb.bin ubuntu/sc-machine:local serve -t -h 0.0.0.0 -c /sc-machine/config/sc-machine.ini
+
+      - name: Check server health
+        run: |
+          pip3 install websocket-client
+          docker run --rm --name server -d -p 8090:8090 -e REBUILD_KB=1 -v ${ls}/kb:/kb -v kb_binary:/kb.bin ubuntu/sc-machine:local serve -h 0.0.0.0 -c /sc-machine/config/sc-machine.ini
+          python3 ./scripts/healthcheck.py -h 0.0.0.0
```

Comment:
`-h` is a wrong option. It's used for help printing. In general, we don't need to specify host here at all, cause we are running server inside docker container and export **8090** port and tell server to listen on all ports; and healthcheck must be running without any arguments.
However, such configuration works on my machine, but fails during CI with `[Errno 104] Connection reset by peer`.